README.md

```diff
@@ -2,10 +2,38 @@
 
 A kubernetes controller that can auto unseal vault pods.
 
+## Labels / Annotations
+
+### StatefulSet
+
+Use the label `vault-unsealer.bakito.net/stateful-set` to unseal a kubernetes vault statefulset in the same namespace
+
+Example: 
+
+```yaml
+  labels:
+    vault-unsealer.bakito.net/stateful-set: vault
+```
+
+### External
+
+Use the label `vault-unsealer.bakito.net/external` flag a secret for usage to handle non-kubernetes vaults. The value specifies the seal check interval duration
+
+Use the annotation `vault-unsealer.bakito.net/external-source` to define a vault where the unseal keys are stored
```

Comment:
could the source also be a vault from the in-namespace StatefulSet?

Reply:
theoretically all combinations could work, external vault as source unseals internal vault, interal vault source unseals external vault.. but then we would need to split the `stateful-set` label into `stateful-set-source` and `stateful-set-target`

Reply:
I would propose to do this as a follow up PR